lib/rendition.js

```diff
@@ -138,6 +93,19 @@ class Rendition {
         }
     }
 
+    get name() {
+        return Rendition.renditionFilename(this.instructions.fmt, this.index);
+    }
+
+    get path() {
+        return this._path || path.join(this.directory, this.name);
+    }
+
+    /** only for unit tests */
+    set path(path) {
```

Comment:
Since it's only for unit tests, I would rename the setter to `_path`, as the `_` is a usual indicator that a method is private in JS.

Reply:
The reason for this change was to not have to change all the test code that does `rendition.path = ...`. I tried other variations (setting through constructor) but they don't work well and would have changed even more code.